scripts/fetch_wikidata_from_everypolitician.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+
+# Fetch a legislature from EveryPolitician, and for all people with a ParlParse
+# ID and Wikidata ID import the Wikidata ID.
+
+from everypolitician import EveryPolitician
+from popolo import Popolo
+from popolo.utils import edit_file
+
+
+def getPersonIdentifierBySchema(person, scheme):
+    if 'identifiers' in person:
+        for identifier in person['identifiers']:
+            if identifier['scheme'] == scheme:
+                return identifier['identifier']
+    return None
+
+
+legislatures_to_fetch = [
+    {
+        'ep_country_slug': 'UK',
+        'ep_legislature_slug': 'Commons'
+    },
+    {
+        'ep_country_slug': 'Scotland',
+        'ep_legislature_slug': 'Parliament'
+    },
+    {
+        'ep_country_slug': 'Northern-Ireland',
+        'ep_legislature_slug': 'Assembly'
+    },
+]
+
+pp_data = Popolo()
+
+for legislature in legislatures_to_fetch:
+
+    ep_country = EveryPolitician().country(legislature['ep_country_slug'])
+    ep_legislature = ep_country.legislature(legislature['ep_legislature_slug'])
+
+    ep_people = ep_legislature.popolo().persons
+
+    print ('{}/{}: Found {} people.'.format(legislature['ep_country_slug'],
+                                            legislature['ep_legislature_slug'],
```

Comment:
It'd be nice to make all this output only output with a verbose flag in case we want to add this to a cron for keeping this up to date.